locale/surveyForm/da.yaml

```diff
@@ -9,15 +9,11 @@ signature:
             label: Send in anonymt
         email:
             desc: >-
-                Skriv under med navn og e-mailadresse tilhørende den der har
-                besvaret, dvs. dig selv, eller den du evt. sender besvarelsen
-                for.
+                Skriv under med navn og e-mailadresse.
```

Comment:
This should have been kept as is. It was only the label that needed to change. The mistake was that the same text was in both places, and now that is still the case, albeit different text. Please revert the change to email.desc.